packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -480,7 +480,8 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         bytes memory ancillaryData,
         bytes32 hash
     ) public override nonReentrant() onlyIfNotMigrated() {
-        uint256 currentRoundId = getCurrentRoundId();
+        uint32 currentRoundId = SafeCast.toUint32(getCurrentRoundId());
```

Comment:
Why did we need to safe cast this? Wouldn't this only fail after 23,518,461 years after 1970?

Reply:
i used safe cast simply for consistency. I've reverted to unsafe casting right at assignment.